feconf.py

```diff
@@ -470,6 +470,12 @@ def get_empty_ratings():
 # A sorted list of default categories.
 DEFAULT_CATEGORIES = sorted(CATEGORIES_TO_COLORS.keys())
 
+#Size of the gravatar in px assigned to the user after signup
+GRAVATAR_SIZE_PX = 150
+
+#Default Identicon for new users
+DEFAULT_IDENTICON_DATA_URL = '/images/avatar/user_blue_72px.png'
```

Comment:
I tried doing that but the data url of any gravatar image is like 5-10 lines long , and I thought including a 5-10 lines long constant in the module would look odd . 
The other option every time we convert that image into the url but that would require additional imports and computations.